Fix release-please workflow

```terminal
npm notice name:          @openscd/core                           
npm notice version:       0.1.1                                   
npm notice filename:      openscd-core-0.1.1.tgz                  
npm notice package size:  22.3 kB                                 
npm notice unpacked size: 90.5 kB                                 
npm notice shasum:        f313b8f8a83d359dcd89cec4d51c945df26053a5
npm notice integrity:     sha512-OwUeWiA6QCMFN[...]fVeloQLBbr5uw==
npm notice total files:   32                                      
npm notice 
npm notice Publishing to https://registry.npmjs.org/ with tag latest and public access
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/@openscd%2fcore - You cannot publish over the previously published versions: 0.1.1.
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy, or
npm ERR! 403 on a server you do not have access to.

npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/2024-02-06T12_05_23_703Z-debug-0.log
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release-please.yml
+++ b/.github/workflows/release-please.yml
@@ -18,7 +18,6 @@ jobs:
         id: release
         with:
           release-type: node
-          package-name: release-please-action
           command: manifest
           token: ${{ secrets.GITHUB_TOKEN }}
       # The logic below handles the npm publication:

--- a/.github/workflows/release-please.yml
+++ b/.github/workflows/release-please.yml
@@ -2,11 +2,13 @@ on:
   push:
     branches:
       - main
+    paths:
+      - "packages/core/**"
 
 permissions:
   contents: write
   pull-requests: write
-  
+
 name: release-please
 jobs:
   release-please:
@@ -15,6 +17,8 @@ jobs:
       - uses: google-github-actions/release-please-action@v4
         id: release
         with:
+          release-type: node
+          package-name: release-please-action
           command: manifest
           token: ${{ secrets.GITHUB_TOKEN }}
       # The logic below handles the npm publication:
@@ -23,7 +27,7 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 18
-          registry-url: 'https://registry.npmjs.org'
+          registry-url: "https://registry.npmjs.org"
         if: ${{ steps.release.outputs.releases_created }}
       - run: npm ci
         if: ${{ steps.release.outputs.releases_created }}

